app/views/donations/index.html.erb

```diff
@@ -4,27 +4,8 @@
     Donation List
   </h2>
   <% if @donations.any? %>
```

Comment:
Please use the viewcomponent conditional rendering method

Reply:
The donation view component represents a single item from the @donations collection instead of the list itself, it feels weird to conditionally render an existing item already being passed as a parameter by comparing if the collection from where that item came from is empty or not.